pypit/arparse.py

```diff
@@ -2090,6 +2090,44 @@ def trace_slits_tilts_order(self, v):
             msgs.error("The argument of {0:s} must be >= 0".format(get_current_name()))
         self.update(v)
 
+    def trace_slits_tilts_yorder(self, v):
+        """ What is the order of the 2D function fitting the tilt shapes in the spectral dimension
+
+        Parameters
+        ----------
+        v : str
+          value of the keyword argument given by the name of this function
+        """
+        v = key_int(v)
+        if v < 0:
+            msgs.error("The argument of {0:s} must be >= 0".format(get_current_name()))
+        self.update(v)
+
+    def trace_slits_tilts_trthrsh(self, v):
+        """ What is the theshold for including an arc line in the tilt analysis
```

Comment:
Typo: `threshold`

Also, it's not immediately obvious (to me) what `trthrsh` stands for in the function name. Maybe it's best to be a little more specific here?